using-mobile-extensions/profile/profiles-event-reference.md

```diff
@@ -77,38 +85,46 @@ The event is used to share the user profile with the other extension. This event
 
 #### Data Payload Definition
 
-| Key | Value Type | Optional | Description |
-| :--- | :--- | :--- | :--- |
+| **Key**         | **Value Type**            | **Optional** | **Description**                                              |
+| :-------------- | :------------------------ | :----------- | :----------------------------------------------------------- |
+| userprofiledata | Map&lt;String, Object&gt; | no           | A map of all the user profile attributes and the attribute values. This map will contain all the user profile attributes (the most updated copy). The consumers of this data may replace the existing user profile copy that they may have with this copy. |
 
+#### 
+
+Here is a code sample for the `USER_PROFILE : RESPONSE_PROFILE` event:
 
-<table>
-  <thead>
-    <tr>
-      <th style="text-align:left">Map</th>
-      <th style="text-align:left">Map</th>
-      <th style="text-align:left">No</th>
-      <th style="text-align:left">
-        <p>A map of all the user profile attributes and the attribute values.</p>
-        <p>This map will contain all the user profile attributes (the most updated
-          copy).</p>
-        <p>The consumers of this data may replace the existing user profile copy
-          that they may have with this copy.</p>
-      </th>
-    </tr>
-  </thead>
-  <tbody></tbody>
-</table>Here is a code sample for the `USER_PROFILE : RESPONSE_PROFILE` event:
 
 ```text
 {
     EventName : "UserProfile Response Event"
     EventType : "com.adobe.eventType.userProfile",
     EventSource : "com.adobe.eventSource.responseProfile",
     EventData: {
+          "userprofiledata":
+               {
                 "userName": "John Doe"
                  "creditPoints": 255,
                  "isPaidUser": true
                }
+          }
 }
 ```
 
+
+
+## Shared State
+
+**EXTENSION_NAME**: `com.adobe.module.userProfile`
+
+The Profile extension shared state is created in the following situations:
+
+- **On Initialization:** After the module is initialized, it will update the shared state by reading the previously set value from persistence.
+- **On Every UserProfile Attribute Create/Update:** Shared stare is update whenever there is a change in the user attribute data either from public API or by other triggers (Rules Engine).
```

Comment:
Editing changes: please update this sentence to read as follows:

"The shared stare is updated when there is a change in the user attribute data from the public API or by other triggers (Rules Engine)."